### DISCUSSION TITLE
"expand up" tooltip obscures disclosure control in source view file headers

### DISCUSSION THREAD

**Discussion by guppy-jpf** [community/community#57198] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

i don't recall this happening before, but recently, it seems every time i want to collapse a diff listing in source comparison view, if i happen to accidentally mouseover the blue elided area thing immediately below the header, an "expand up" tooltip appears, obscuring the control i am trying to target.  that wouldn't be an issue if the tooltip would actually disappear, but it remains even after the cursor leaves the blue region.  it's kind of driving me crazy.

(firefox 113.0.1)

<img width="148" alt="Screenshot 2023-06-05 at 13 32 58" src="https://github.com/community/community/assets/69841130/d81846ca-56f8-43f2-a6b4-215eaaa6c471">

**Comment by di** [comment, marked as answer]
This seems like a duplicate of https://github.com/orgs/community/discussions/54674
